tests/validation/rules/requiredHmsIf.spec.ts

```diff
@@ -0,0 +1,92 @@
+import { requiredHmsIf } from 'sefirot/validation/rules'
+import { type RequiredIfCondition } from 'sefirot/validation/validators'
+
+describe('validation/rules/requiredHmsIf', () => {
+  function getTestCasesForAllTypes() {
+    return [
+      { value: { hour: '01', minute: '02', second: '03' }, expected: { required: true, unrequired: true } },
```

Comment:
As commented on the validator, I don't think it makes sense to have `required: false` condition.

If condition is false, then everything should return true 🤔

Reply:
Fixed: 21c1168